apps/charterafrica/src/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,84 @@
+import RichTypography from "@/commons-ui/core/RichTypography";
+import Section from "@/commons-ui/core/Section";
+import { Link } from "@commons-ui/next";
+import { Button } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { secondary, neutral } from "@/charterafrica/colors";
+import BackgroundBox from "@/charterafrica/components/BackgroundBox";
+import RichText from "@/charterafrica/components/RichText";
+
+const ErrorPage = React.forwardRef(function Error(props, ref) {
+  const {
+    statusCode,
+    description = [],
+    title = "",
+    createdAt,
+    updatedAt,
+    blockType,
+    action,
+    ...other
+  } = props;
+  return (
+    <BackgroundBox
+      {...other}
+      bgcolor={secondary[50]}
+      display="flex"
+      alignItems="center"
+      sx={{
+        backgroundImage: {
+          md: `url(/images/notFound.png)`,
+        },
+        height: "100%",
+        width: "100%",
+        position: { md: "absolute", xs: "relative" },
+        backgroundSize: "50% 100%",
+        ...props?.sx,
+      }}
+      ref={ref}
+    >
+      <Section sx={{ p: 10 }}>
+        <RichTypography color={neutral[900]} textAlign="center" variant="h1">
+          {statusCode}
+        </RichTypography>
+        <RichTypography
+          sx={{ mt: 2.5, textTransform: "uppercase" }}
```

Comment:
```suggestion
          mt={2.5}
          textTransform="uppercase"
```

Typography component support these props directly.